services/common/snmp/ntcip1218/rsu_location.py

```diff
@@ -0,0 +1,98 @@
+import subprocess
+import logging
+import common.snmp.snmpcredential as snmpcredential
+import common.snmp.snmperrorcheck as snmperrorcheck
+
+
+def convert_location_value(val):
```

Comment:
suggestion: refactor to remove magic strings. ex:
```
UNKNOWN_LATITUDE = "900000001"
UNKNOWN_LONGITUDE = "1800000001"
DEGREE_DIVISOR = 10000000.0

def convert_location_value(val):
    """
    Converts the SNMP NTCIP-1218 value for latitude or longitude into a float in degrees.
    """
    numerical = val.split(" ")[0]
    if numerical == UNKNOWN_LATITUDE:
        return None
    elif numerical == UNKNOWN_LONGITUDE:
        return None
    else:
        try:
            return float(numerical) / DEGREE_DIVISOR
        except ValueError:
            logging.error(f"Invalid value for latitude/longitude: {val}")
    return None
```

Reply:
Addressed with your suggestion.